### PR TITLE
subscription_info module

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: plugins/modules/subscription_info.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - theforeman.foreman.foreman.infomodule
+'''
+
+EXAMPLES = '''
+- name: "Show a subscription"
+  theforeman.foreman.subscription_info:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    name: "Red Hat Satellite Infrastructure Subscription"
```
Comment (ehelms) [inline plugins/modules/subscription_info.py:42]: No organization context is needed to fetch sub info?

File: plugins/modules/subscription_info.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- name: "Show all subscriptions with a certain name"
+  theforeman.foreman.subscription_info:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    search: name="Red Hat Satellite Infrastructure Subscription"
```
Comment (ehelms) [inline plugins/modules/subscription_info.py:49]: No organization context is needed to fetch sub info?

File: plugins/modules/subscription_info.py (hunk 1 above)
Comment (evgeni) [reply plugins/modules/subscription_info.py:42]: According to https://theforeman.org/plugins/katello/3.18/api/index.html - no.

Comment (ehelms) [reply plugins/modules/subscription_info.py:42]: Can we have one or all of the tests reflect that so we are testing the example? (e.g. drop https://github.com/theforeman/foreman-ansible-modules/pull/1223/files#diff-26c401d211772be08e88a4e26eb7f42fc1c21f16b49ace60767e3113a237b49eR29)

Comment (evgeni) [reply plugins/modules/subscription_info.py:42]: dropped from all tests

**Review by ehelms — APPROVED** [review]

**Pull request merged**